index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+    <title>
+        youtube home page
+    </title>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="index.css">
+    <script src="https://kit.fontawesome.com/cce6e1325c.js" crossorigin="anonymous"></script>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+<body>
+    <header>
+        <nav>
+            <div id="nav">
+                <label>
+                <img src="/image/icons8-menu-26.png" alt="menu"> 
+                <img src="/image/logo.png" id="logo" alt="logo">
```

Comment:
The icons are not showing, to fix it you guys can put `./` at the beginning of the image path like this `src="./image/icons8-menu-26.png"`